src/Azure/Shared/Storage/AzureStorageUtils.cs

```diff
@@ -25,10 +26,28 @@ namespace Orleans.Streaming.AzureStorage
 namespace Orleans.Streaming.EventHubs
 #elif TESTER_AZUREUTILS
 namespace Orleans.Tests.AzureUtils
+#elif ORLEANS_TRANSACTIONS
+namespace Orleans.Transactions.AzureStorage
 #else
 // No default namespace intentionally to cause compile errors if something is not defined
 #endif
 {
+    /// <summary>
+    /// Constants related to Azure table storage.
+    /// </summary>
+    internal static class AzureTableConstants
+    {
+        /// <summary>
+        /// ETag of value "*" to match any etag for conditional table operations (update, nerge, delete).
+        /// </summary>
+        public const string ANY_ETAG = "*";
+
+        public const string PKProperty = "PartitionKey";
```

Comment:
nameof(TableEntity.PartitionKey) ? for compile time safe

Reply:
or `TableConstants.PartitionKey`

https://docs.microsoft.com/en-us/dotnet/api/microsoft.windowsazure.storage.table.protocol.tableconstants?view=azure-dotnet